site_root/input/resources/profiles/StructureDefinition-kfdrc-research-study-no-phi.json

```diff
@@ -0,0 +1,33 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "kfdrc-research-study-no-phi",
+  "url": "http://fhir.kids-first.io/StructureDefinition/kfdrc-research-study-no-phi",
+  "version": "0.1.0",
+  "name": "kfdrc-research-study-no-phi",
+  "title": "Kids First DRC ResearchStudy without Protected Health Information",
+  "status": "draft",
+  "fhirVersion": "4.0.0",
+  "kind": "resource",
+  "abstract": false,
+  "type": "ResearchStudy",
+  "baseDefinition": "http://fhir.kids-first.io/StructureDefinition/kfdrc-research-study",
+  "derivation": "constraint",
+  "differential": {
+    "element": [
+      {
+        "id": "ResearchStudy",
+        "path": "ResearchStudy"
+      },
+      {
+        "id": "ResearchStudy.contact",
+        "path": "ResearchStudy.contact",
+        "max": "0"
+      },
+      {
+        "id": "ResearchStudy.period",
```

Comment:
I don't think this is PHI since its in reference to the research study and not a patient

Reply:
Hmm. I thought the period specifies the duration of a study and may be a clue to identify patients?

Reply:
@allisonheath ?

Reply:
Ok yea, it looks like it includes a start/end date for a study: https://www.hl7.org/fhir/researchstudy-definitions.html#ResearchStudy.period

I don't think we typically get this information, if we do it's on the granularity of year (maybe month). In theory it could correspond to the IRB approval/close out date, which isn't PHI. So bottom line, I think it's fine to keep in.

Reply:
That PHI profile only had period as a differential, so I just retired the profile!